packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -543,6 +543,50 @@ class UsersRepo extends AbstractRepo<"Users"> {
     `, [limit])
   }
 
+   async getUsersForInactiveSummaryEmail(limit: number): Promise<(DbUser & {
+     fetchActivitySince: Date,
+   })[]> {
+    return this.getRawDb().manyOrNone(`
+      -- UsersRepo.getUsersForInactiveSummaryEmail
+      SELECT
+        u.*,
+        GREATEST(
+          u."inactiveSummaryEmailSentAt",
+          rs."max_last_updated"
+        ) "fetchActivitySince"
+      FROM public."Users" AS u
+      LEFT JOIN (
+        SELECT "userId", MAX("lastUpdated") AS max_last_updated
+        FROM "ReadStatuses"
+        WHERE "isRead" IS TRUE
+        GROUP BY "userId"
+      ) AS rs ON u._id = rs."userId"
+      WHERE
+        (
+          u."inactiveSummaryEmailSentAt" IS NULL
+          OR u."inactiveSummaryEmailSentAt" < CURRENT_TIMESTAMP - INTERVAL '50 days'
```

Comment:
This means if someone never comes back they'll get an email every 50 days forever right? (I think this is ok, it's still a very low rate of emails)